docs/development/releasing.rst

```diff
@@ -412,22 +421,12 @@ clean-up tasks to finalize the process.
 Post-Release procedures
 -----------------------
 
-#. If this is a release of the current release (i.e., not an LTS supported along
-   side a more recent version), update the "stable" branch to point to the new
-   release::
+#. Update the "stable" branch to point to the new release::
```

Comment:
Do we still need stable branch?

Reply:
No I think we can remove this if we set up the automatic updating of stable on RTD

Reply:
It sounds like this should work by default *but* I guess we will need to remove the stable branch in our repo and probably also wipe stable in RTD after we've done that and remove the explicitly specified stable version.